lib/checkPullRequestReview.js

```diff
@@ -381,9 +408,16 @@ const handleResponseToReview = async (context) => {
   }
 };
 
+const handleLabels = async(context)=>{
+  const allOpenPullRequests = await getAllOpenPullRequests(context);
+  allOpenPullRequests.map((pullRequest)=>{
+    handleChangesRequested(context, pullRequest, pullRequest.base);
+  });
+};
```

Comment:
This is redundant. 
The `handleChangesRequested` function will always be called whenever a pull request has changes requested.

Reply:
Ok Sir, Will remove that.